db/migrate/20230311022744_create_books.rb

```diff
@@ -0,0 +1,8 @@
+class CreateBooks < ActiveRecord::Migration[7.0]
+  def change
+    create_table :books do |t|
+
```

Comment:
It looks like you ran `rails g migration books` and then you ran `rails g model Book`. 

As a result, you have 2 migrations that create the `books` table. 

My suggestions:
* Let's keep the `db/migrate/20230311022744_create_books.rb`
* Copy the `def change` body into this file
* Delete the `db/migrate/20230311021131_books.rb`